Angular/src/app/tooltip/tooltip.component.ts

```diff
@@ -0,0 +1,20 @@
+import {
+  Component, Input, Output, EventEmitter,
+} from '@angular/core';
+import { ClickEvent } from 'devextreme/ui/button';
+import { Appointment } from 'devextreme/ui/scheduler';
+
+@Component({
+  selector: 'Tooltip',
+  templateUrl: './tooltip.component.html',
+})
+export class TooltipComponent {
+  @Input()
+    appointmentData!: Appointment;
```

Comment:
Let's write it in one line:
```suggestion
  @Input() appointmentData!: Appointment;
```